worker/src/main/java/gov/cms/ab2d/worker/config/WorkerConfig.java

```diff
@@ -39,6 +43,19 @@
     @Autowired
     private JobHandler handler;
 
+    @Bean("bfd-client")
+    public Executor bfdThreadPoolTaskExecutor() {
+        final ThreadPoolTaskExecutor taskExecutor = new ThreadPoolTaskExecutor();
+        taskExecutor.setCorePoolSize(5);
```

Comment:
Let's make these 3 pool parameters configurable via `application.properties` (can be done in a sub-task)

Reply:
ok. I will make this change in the next sub-task for this story.

Reply:
done